docker-entrypoint.sh

```diff
@@ -71,6 +71,6 @@ wait-for-url() {
 }
 
 # Dashboard service
-(cd /dashboard; wait-for-url localhost:8088/v1/version && yarn start:docker) &
+(cd /dashboard; wait-for-url localhost:8088/v1/version && NEXT_PUBLIC_UID=$(cat /proc/sys/kernel/random/uuid) yarn start:docker) &
```

Comment:
Oh wow, I had no idea The Linux kernel itself sprouted a UUID generator. That's cool :)

Reply:
Yes, I also learned the today!